content/events/2021-online-unconference/_index.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Nordic-RSE online unconference"
+description = "Nordic RSE – Research Software Engineers in the nordics – Unconference 29–30.June #NordiRSEunconf"
++++
+
+# Nordic RSE Unconference 2021
+
+We are planning an online unconference on June 29–30. The program will consist of your contributions and we encourage you to submit a quick abstract for a discussion topic, talk, demonstration or any other type of program you would like to run.
+
+## Dates: June 29-30, 13-16 CET (3 + 3 hours)
+
+## Format and goal and scope
+
+To bring the community of research software engineers together in 2021,
+to exchange ideas and solutions and challenges we plan an informal unconference.
+In the unconference format we encourage participants to shape the program and
+we will provide support and infrastructure for that.
+
+This event is not only limited to the Nordics and not only limited to participants
+who are or identify as RSEs but anybody interested in the interface of research and software and data is welcome.
+
+
+## What contributions we welcome
+
+In principle we welcome any contributions which respect the Code of Conduct and in principle of any length. We will try to arrange a program based on your input.
+
+Here are some examples of possible contributions:
+- Discussion topics
+- Demonstrations (tools you like, software you wrote)
+- Quick workshops or reprohacks (only 3 hours in a day)
+- Talks
+- Workshops and lessons: teach us something new
+- Can also be something unfinished and almost working where you would like to get feedback on
+
+## How to contribute ideas
+
+- You can submit a short abstract as a [GitHub issue](https://github.com/nordic-rse/nordic-rse.github.io/issues/new).
+  <!-- - You can also email us at ... Note that your name and abstract will be published.
+  What is our email?
+  -->
+- But you can also pitch an idea at the event but the earlier we know the easier scheduling for us
+- There is no formal review process but we provide support giving feedback on issues.
+- Contributions should not violate CoC: "All participants, speakers, organizers, and volunteers at this event are required to conform to the following code of conduct."
+
+
+## Tentative format
+
+- Day 1
+  - 13:00 : Pitch topics + Unconference session
+  - 14:20 : Break and free discussion
+  - 14:40 : Unconference session
```

Comment:
I would separate parts where we have a program (based on prior submissions) and part where anyone can come with an idea on the day.

Reply:
I think we should do as little planning as possible. The unplanned 'unconference sessions' worked well in December.

But it is also helpful for potential participants to see some ideas, so getting some submissions before the event is also a good idea.

I would not separate these. This would limit the time available for same-day ideas.